src/TabBar.js

```diff
@@ -155,6 +142,12 @@ export default class TabBar<T: *> extends React.Component<Props<T>, State> {
   _panXListener: string;
   _offsetXListener: string;
 
+  _resetState = () => {
+    this.props.scrollEnabled && this._stopTrackingPosition();
+    this.state.scrollAmount.setValue(0);
```

Comment:
How does this fix the issue? If the current tab is not the first one, shouldn't this value be set accordingly?